babel.config.json

```diff
@@ -0,0 +1,12 @@
+{
+  "presets": [
+    [
+      "@babel/env",
+      {
+        "targets": {
```

Comment:
Since you have started this file, I'd say that it should serve as a true source of which browsers are supported, and I'd add info about other browsers here as well, even though it will likely not affect babel's behaviour.